### ISSUE TITLE
Question: modeling features of objects (material, subject, type) as characteristics of artists

### ISSUE THREAD

**Issue by rpiening** [linked-art/linked.art#496]
Hi all,

I'm working on the same dataset with biographical information of artists as @sdevalk , using the [Actor patterns](https://linked.art/model/actor/) as foundation. 

This dataset contains information about the materials and genres an artist is known to (have) worked in, as well as the type of objects they make/made. For example:

Arnao de Bruselas
role: carver
subject/genre: christian religious scenes
material: wood
types of objects: altar, choir stall

Typically you might want to infer this kind of information from art objects linked to the artist. However, the source of this knowledge isn't always (extant) objects, but might come from a textual source that doesn't otherwise mentions specific objects. As such, it can be valuable to model this information onto the artist themselves.

Do you have recommendations on how to model this, either within the Linked Art model, in CIDOC-CRM, or using something else?

Many thanks.

**Comment by Habennin** [comment]
These are kind of categorical statements about the individual. You don't know particular objects but you do have information about the kinds of material worked with, objects created etc. 

The Canadian Heritage Information Network has wrestled with modelling some of this data for the project Makers in Canada. You can see their target semantic model here:

https://chin-rcip.github.io/collections-model/en/target-model/current/introduction

Because CRM does not deal much with prosopographic data there aren't very convenient direct relations from person to different kinds of pursuit. In my opinion this should be raised to the CRM SIG as a problem.

There is no property E21 worked with material of type E57 Material, E21 created objects of type E55 .... these are very legitimate and repeating kinds of documentation and so having properties for them in principle makes sense.

Since there are no properties for them at the moment however one work around is to use the Linked.Art type of type pattern

So you say E21 p2 has type E55 (wood) p2 has type -> E55 (material interest)

The problem with this pattern is that you assert that wood is always a material interest and not that it is in particular a material interest of this artist in particular.

Another option is to use the class F51 Pursuit from FRBR2.4 which is an event, but an event covering an indeterminate period of time in which someone is occupied with something. This can be used to hang the interests off of like so:

E21->p14i->E51->R59 had typical subject (was typical subject of): E1 CRM Entity

Then you would fill the E1 with your subject/genre, material and object type data using type and metatype.

E21->p14i->E51->R59 had typical subject (was typical subject of): E55 (wood)
E21->p14i->E51->R59 had typical subject (was typical subject of): E55 (wood) ->p2 ->E55 (material interest)

Again there is a bit of a glitch semantically in the type of type pattern but it'll do.

See FRBR2.4 here:
https://cidoc-crm.org/frbroo/sites/default/files/FRBRoo_V2.4.pdf

**Comment by cbutcosk** [comment]
If you're already in linked.art another approach might be to use an Activity that is classified with a term expressing their genre/format. This is similar to [Active Periods](https://linked.art/model/actor/#active-dates).

We used a pattern like this in the data for [this site](https://missouriartists.org/artist/). The Person object has `Activity` nodes that carry a guide classification to find the genre Activity nodes and also the role / genre / format classification itself:
```json
{
"id": "ex:artist/1",
"type": "Person",
"carried_out": [ 
	{ "classified_as": [ "aat:miniaturist", "aat:genre" ], "type": "Activity", "id": "ex:artist/1/genres/miniaturist" }, 
	{ "classified_as": [ "aat:sculptor", "aat:genre" ], "type": "Activity", "id": "ex:artist/1/genres/sculptor" }
 ]
}
```
This worked fine, though if I were redoing things I'd probably make it completely look like Active Periods, with `carried_out` pointing to an `Activity` node classified `aat:genre` (or, whatever classifier) partitioned with the individual role / genre nodes. In terms of evidence `Activity` gives you `motivated_by`/`motivates`, though `documents` is also there if you're willing to handle its somewhat awkward domain/range constraints (it requires a crm:E31_Document on one end)?

**Comment by azaroth42** [comment]
I agree with @cbutcosk about the Activity pattern.

To try and express the semantics a little differently:

The artist's professional work involved objects which were [classified as] wood, altars, choir stalls, religious scenes.
(And carver can go as the classification of the activity itself, it's a carving sort of activity but it's not an altar sort of activity)

So I would use crm:P125_used_object_of_type on the professional activity pattern already in linked art.  The property is described as:
> This property associates an instance of E7 Activity to an instance of E55 Type, which classifies an instance of E70 Thing used in an instance of E7 Activity, **when the specific instance is either unknown** or not of interest, such as use of "a hammer".
(emphasis added)

**Comment by rpiening** [comment]
Many thanks everyone for your insightful comments. We're currently discussing how to best implement these patterns on our dataset and might get back to you on that. In the meantime, your help has been much appreciated.

**Comment by rpiening** [comment]
> We used a pattern like this in the data for [this site](https://missouriartists.org/artist/).

@cbutcosk Thank you, this is a very interesting website with a similar type of data to our dataset. I'm looking for the underlying linked data, is it available?

**Comment by rpiening** [comment]
> So I would use crm:P125_used_object_of_type on the professional activity pattern already in linked art. The property is described as:
> 
> > This property associates an instance of E7 Activity to an instance of E55 Type, which classifies an instance of E70 Thing used in an instance of E7 Activity, **when the specific instance is either unknown** or not of interest, such as use of "a hammer".
> > (emphasis added)

@azaroth42 Thank you for the suggestion of this property! It fits the use of a material well, but we found it slightly difficult to imagine an 'altar' being used in the making of an altar. Perhaps crm:P186_produced_thing_of_product_type would be a better fit with regard to products - though that leaves an open question of what to do with genre. What are your thoughts?

**Comment by azaroth42** [comment]
@rpiening @sdevalk Did you ever implement anything along these lines? If yes, how did it work out? If no, do you miss the lack of functionality or not?

**Comment by rpiening** [comment]
@azaroth42 Yes we did! We used the activity pattern suggested above. We would've sorely missed it if we hadn't modeled this information, it's not only crucial for the detail information on artists but also part of our Elastic facets.
Thanks to the discussion above, we landed on the following:

```
<https://data.rkd.nl/artists/17999>

#Pattern: https://linked.art/model/actor/#active-dates
#The artist's general period of activity
    crm:P14i_performed [

        a crm:E7_Activity ;

        crm:P2_has_type <http://vocab.getty.edu/aat/300393177> ;

        crm:P67i_is_referred_to_by [

            a crm:E33_Linguistic_Object ;

            crm:P2_has_type <http://vocab.getty.edu/aat/300435416> ;

            # werkzame_periode_bijzonderheid
            crm:P190_has_symbolic_content "ca. 1940-2005" ;

            # There is no crm:P72_has_language here - the language is unknown
        ] ;

        crm:P4_has_time-span [

            a crm:E52_Time-Span ;

            # werkzame_periode_begin
            crm:P82a_begin_of_the_begin "1940"^^xsd:gYear ;

            # werkzame_periode_eind
            crm:P82b_end_of_the_end "2005"^^xsd:gYear ;
        ] ;


        # Pattern: https://github.com/linked-art/linked.art/issues/496#issuecomment-1213307292
        # materiaal_lref
        # This artist has several references. Only some are listed here, for illustration.
        crm:P125_used_object_of_type <https://data.rkd.nl/thesaurus/37107>,
                                     <https://data.rkd.nl/thesaurus/67763> ;

        # onderwerp_werkvorm_lref
        # This artist has several references. Only some are listed here, for illustration.
        crm:P125_used_object_of_type <https://data.rkd.nl/thesaurus/65205>,
                                     <https://data.rkd.nl/thesaurus/63258> ;

        # product_lref
        crm:P125_used_object_of_type <https://data.rkd.nl/thesaurus/74253> ;
    ] ;
``` 
(see it in action [here](https://rkd.triply.cc/rkd/RKD-Knowledge-Graph/browser?resource=https%3A%2F%2Fdata.rkd.nl%2Fartists%2F17999))

We've used P125 for all three options (material, subject and product). Towards the frontend, the distinction is made based on the `skoks:inScheme` of the thesaurus term, like so:
```      
    # ./materiaal [0..n]
    UNION {
        _:performed crm:P125_used_object_of_type ?materiaal .
        ?materiaal skos:prefLabel ?materiaal_label ;
                   skos:inScheme scheme:MATERTECHN .
        FILTER NOT EXISTS {?materiaal skos:inScheme scheme:OBJALG}
    }
```
The resulting [front end page](https://research.rkd.nl/en/detail/https%3A%2F%2Fdata.rkd.nl%2Fartists%2F17999) shows the three distinct fields.

A major challenge that emerged from combining CMS fields and splitting them on `skos:inScheme`, is that it wholly depends on the schemes not overlapping. In our current Thesaurus structure, this isn't 100% guaranteed. Occasionally, a term that was in one field in the source data will be displayed in two fields on the front end. As we've used this trick in various other places where our own datamodel was more granular than LA/CIDOC-CRM, there might be a productive discussion to be had about whether LA can make recommendations on how to disambiguate semantic nuances like this.

Does this long-winding reply answer your question? ;)

**Comment by azaroth42** [comment]
Proposal:  add `used_object_of_type` to the generic activity pattern to allow this.

**Comment by azaroth42** [comment]
Discussion on 2026-09-02: Only gets 70% of the way as Thing doesn't include concepts, only Symbolic and Physical objects.
Defer to 1.2 to work on a closer to 100% solution and leave the `used_object_of_type` as an extension for now.
